README.md

```diff
@@ -156,6 +156,89 @@ response:
 }
 ```
 
+### Create User account
+
+```js
+const rf = require('routefusion-sdk').Instance();
+
+let body = {
+  name_on_account: 'Bob TheBuilder',
+  account_number: '123',
+  routing_number: '1234',
+  currency: 'USD',
+  bank_country: 'US'
+};
+
+rf.createUserAccounts(body)
```

Comment:
Can we rename this `createUserAccount`? All of the other APIs in this SDK use a singular resource naming convention when dealing with a single resource, e.g. create, get (one record)

Reply:
Looks like all three functions return only one object, so they should all be singular names.

Reply:
i kept the names in line with the endpoints but i can change them. The one endpoint returns an array of 1 item. The reason being to confirm to the future of multiple accounts. I will update the singular endpoint

Reply:
updated

Reply:
Ok, just need to update the README to match now, it still has the plurals in it.

Reply:
thanks

Reply:
done